Maven ITs fail out-of-the-box

### Expected Behaviour

Running `mvn clean install` should be successful.

### Actual Behaviour

Running `mvn clean install` fails due to the Maven plug-in ITs not running properly.

```
[INFO] --- maven-invoker-plugin:3.2.2:verify (integration-test) @ aemanalyser-maven-plugin ---
[INFO] -------------------------------------------------
[INFO] Build Summary:
[INFO]   Passed: 0, Failed: 2, Errors: 0, Skipped: 0
[INFO] -------------------------------------------------
[ERROR] The following builds failed:
[ERROR] *  content-package-files-test/pom.xml
[ERROR] *  wknd-test/pom.xml
[INFO] -------------------------------------------------
```

The error that is reported is

```
[ERROR] Failed to execute goal com.adobe.aem:aemanalyser-maven-plugin:1.4.0-SNAPSHOT:project-analyse (aem-analyser) on project aemanalyser-maven-plugin-it-content-package-files-test: Failed to resolve version for com.adobe.aem:aemanalyser-maven-plugin:jar:RELEASE -> [Help 1]
```

This is with 
```
$ git describe HEAD
aemanalyser-maven-plugin-1.3.0-9-gde8887f
```

--- a/aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/VersionUtil.java
+++ b/aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/VersionUtil.java
@@ -26,6 +26,8 @@ import org.eclipse.aether.RepositorySystem;
 import org.eclipse.aether.RepositorySystemSession;
 import org.eclipse.aether.artifact.Artifact;
 import org.eclipse.aether.artifact.DefaultArtifact;
+import org.eclipse.aether.repository.RemoteRepository;
+import org.eclipse.aether.repository.RepositoryPolicy;
 import org.eclipse.aether.resolution.VersionRequest;
 import org.eclipse.aether.resolution.VersionResolutionException;
 import org.eclipse.aether.resolution.VersionResult;
@@ -214,7 +216,7 @@ public class VersionUtil {
         return null;
     }
 
-    String getLatestVersion( final Dependency dependency )
+    String getLatestVersion(final Dependency dependency)
         throws MojoExecutionException {
 
         if ( this.isOffline ) {
@@ -226,14 +228,28 @@ public class VersionUtil {
                 dependency.getClassifier(),
                 artifactHandlerManager.getArtifactHandler(dependency.getType()).getExtension(),
                 "RELEASE"); // this refers to the latest release version
-
-        VersionRequest versionRequest = new VersionRequest(artifact, PLUGIN_TYPE.equals(dependency.getType()) ? project.getRemotePluginRepositories() :  project.getRemoteProjectRepositories(), null);
+        
+        List<RemoteRepository> repositories = getRemoteRepositoriesWithUpdatePolicy(
+        		PLUGIN_TYPE.equals(dependency.getType()) ? project.getRemotePluginRepositories() :  project.getRemoteProjectRepositories(),
+        		RepositoryPolicy.UPDATE_POLICY_ALWAYS);
+        VersionRequest versionRequest = new VersionRequest(artifact, repositories, null);
         try {
             VersionResult result = repoSystem.resolveVersion(repoSession, versionRequest);
             return result.getVersion();
         } catch (VersionResolutionException e) {
             throw new MojoExecutionException(e.getMessage(), e);
         }
+    }
+    
+    private List<RemoteRepository> getRemoteRepositoriesWithUpdatePolicy(List<RemoteRepository> repositories, String updatePolicy) {
+    	List<RemoteRepository> newRepositories = new ArrayList<>();
+    	for (RemoteRepository repo : repositories) {
+    		RemoteRepository.Builder builder = new RemoteRepository.Builder(repo);
+    		RepositoryPolicy newPolicy = new RepositoryPolicy(repo.getPolicy(false).isEnabled(), updatePolicy, repo.getPolicy(false).getChecksumPolicy());
+    		builder.setPolicy(newPolicy);
+    		newRepositories.add(builder.build());
+    	}
+    	return newRepositories;
     }
 
     /**

--- a/aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/VersionUtil.java
+++ b/aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/VersionUtil.java
@@ -228,10 +228,8 @@ public class VersionUtil {
                 dependency.getClassifier(),
                 artifactHandlerManager.getArtifactHandler(dependency.getType()).getExtension(),
                 "RELEASE"); // this refers to the latest release version
-        
         List<RemoteRepository> repositories = getRemoteRepositoriesWithUpdatePolicy(
-        		PLUGIN_TYPE.equals(dependency.getType()) ? project.getRemotePluginRepositories() :  project.getRemoteProjectRepositories(),
-        		RepositoryPolicy.UPDATE_POLICY_ALWAYS);
+                PLUGIN_TYPE.equals(dependency.getType()) ? project.getRemotePluginRepositories() : project.getRemoteProjectRepositories(), RepositoryPolicy.UPDATE_POLICY_ALWAYS);
         VersionRequest versionRequest = new VersionRequest(artifact, repositories, null);
         try {
             VersionResult result = repoSystem.resolveVersion(repoSession, versionRequest);
@@ -240,16 +238,16 @@ public class VersionUtil {
             throw new MojoExecutionException(e.getMessage(), e);
         }
     }
-    
+
     private List<RemoteRepository> getRemoteRepositoriesWithUpdatePolicy(List<RemoteRepository> repositories, String updatePolicy) {
-    	List<RemoteRepository> newRepositories = new ArrayList<>();
-    	for (RemoteRepository repo : repositories) {
-    		RemoteRepository.Builder builder = new RemoteRepository.Builder(repo);
-    		RepositoryPolicy newPolicy = new RepositoryPolicy(repo.getPolicy(false).isEnabled(), updatePolicy, repo.getPolicy(false).getChecksumPolicy());
-    		builder.setPolicy(newPolicy);
-    		newRepositories.add(builder.build());
-    	}
-    	return newRepositories;
+        List<RemoteRepository> newRepositories = new ArrayList<>();
+        for (RemoteRepository repo : repositories) {
+            RemoteRepository.Builder builder = new RemoteRepository.Builder(repo);
+            RepositoryPolicy newPolicy = new RepositoryPolicy(repo.getPolicy(false).isEnabled(), updatePolicy, repo.getPolicy(false).getChecksumPolicy());
+            builder.setPolicy(newPolicy);
+            newRepositories.add(builder.build());
+        }
+        return newRepositories;
     }
 
     /**

